components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/BrowserTabsTray.kt

```diff
@@ -23,7 +23,8 @@ class BrowserTabsTray @JvmOverloads constructor(
     context: Context,
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0,
-    val tabsAdapter: TabsAdapter = TabsAdapter()
+    val tabsAdapter: TabsAdapter = TabsAdapter(),
+    val layout: LayoutManager = GridLayoutManager(context, 2)
```

Comment:
nit: I don't think you need a `val` here since you are using the layout manager in the `init` block.

`val` holds a reference to the object in this class but since we don't need to call it anywhere else, it gets consumed immediately.